### DISCUSSION TITLE
documentation can't be downloaded

### DISCUSSION THREAD

**Discussion by tanto-grs** [IndySockets/Indy#588] [Q&A] [answered]
Hey guys,
i have the problem, that the documentation can't be downloaded. Not the .zip or .7z files
is it possible to repair this?

best
regards

**Comment by rlebeau** [comment]
Can you be more specific? Which download exactly is not working?  The [HTMLHelp `.zip` file](https://github.com/IndySockets/Indy/files/7782730/IndyDocs_10.1.5.0_HtmlHelp.zip) on this repo works fine for me.

**Comment by tanto-grs** [comment]
hey,
i mean the pdf as zip or 7z

**Comment by rlebeau** [comment]
What PDF are you referring to?  What is the EXACT url you are trying to download that is not working?

**Comment by fabiosabatinebock** [comment]
> What PDF are you referring to? What is the EXACT url you are trying to download that is not working?

All the links in the download section is broken, there's no link working.

**Comment by rlebeau** [comment, marked as answer]
If you are referring to the download links on https://www.indyproject.org/documentation/ then yes, they are all broken. Please see this blog article about that: https://www.indyproject.org/2021/02/10/links-to-old-indy-website-pages-are-currently-broken/

Also see https://github.com/IndySockets/Indy/wiki/Documentation/ on this repo.
